bills/views.py

```diff
@@ -1,5 +1,31 @@
+# pylint: disable=too-many-ancestors,no-member, unused-argument
 """Views for bills application."""
 
-# from django.shortcuts import render
+from rest_framework import viewsets
 
-# Create your views here.
+
+def create_viewset(serializer, model, nested):
```

Comment:
Nice piece of code, but do something with view names:
![obraz](https://user-images.githubusercontent.com/3868440/71125506-57a7d400-21e7-11ea-8c79-fb7ec31efd3e.png)
It uses the same name for every instance no matter what is the model.

Reply:
Done by adding:
```python
def __init__(self, *args, **kwargs):
    self.__class__.__name__ = "{} view".format(model.__name__)
    super().__init__(*args, **kwargs)
```